test/test_document_store.py

```diff
@@ -1633,6 +1636,146 @@ def test_DeepsetCloudDocumentStore_query(deepset_cloud_document_store):
+
+@responses.activate
+def test_DeepsetCloudDocumentStore_lists_evaluation_set_names(deepset_cloud_document_store):
+    response_evaluation_set = {
+        "evaluation_set_id": uuid4(),
+        "name": DC_TEST_INDEX,
+        "created_at": "2022-03-22T13:40:27.535Z",
+        "matched_labels": 2,
+        "total_labels": 10,
+    }
+    if MOCK_DC:
+        responses.add(
+            method=responses.GET,
+            url=f"{DC_API_ENDPOINT}/workspaces/default/evaluation_sets",
+            status=200,
+            body={"data": [response_evaluation_set], "has_more": False, "total": 1},
+        )
+    else:
+        responses.add_passthru(DC_API_ENDPOINT)
+
+    evaluation_sets = deepset_cloud_document_store.get_evaluation_sets()
+    assert evaluation_sets == [response_evaluation_set]
+
+
+@responses.activate
+def test_DeepsetCloudDocumentStore_fetches_lables_for_evaluation_set(deepset_cloud_document_store):
+    if MOCK_DC:
+        responses.add(
+            method=responses.GET,
+            url=f"{DC_API_ENDPOINT}/workspaces/default/evaluation_sets",
+            status=200,
+            body=[
+                {
+                    "label_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+                    "query": "What is berlin?",
+                    "answer": "biggest city in germany",
+                    "answer_start": 0,
+                    "answer_end": 0,
+                    "meta": {},
+                    "context": "Berlin is the biggest city in germany.",
+                    "external_file_name": "string",
+                    "file_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+                    "state": "Label matching status",
+                    "candidates": "Candidates that were found in the label <-> file matching",
+                }
+            ],
+        )
+    else:
+        responses.add_passthru(DC_API_ENDPOINT)
+
+    labels = deepset_cloud_document_store.get_all_labels(label_index=DC_TEST_INDEX)
+    assert labels == [
+        Label(
+            query="What is berlin?",
+            document=Document(content="Berlin is the biggest city in germany."),
+            is_correct_answer=True,
+            is_correct_document=True,
+            origin="user-feedback",
+            answer=Answer("biggest city in germany"),
+            id="3fa85f64-5717-4562-b3fc-2c963f66afa6",
+            no_answer=False,
+            pipeline_id=None,
+            created_at=None,
+            updated_at=None,
+            meta={},
+            filters={},
+        )
+    ]
+
+
+@responses.activate
+def test_DeepsetCloudDocumentStore_fetches_lables_for_evaluation_set_raises_deepsetclouderror_when_nothing_found(
+    deepset_cloud_document_store,
+):
+    if MOCK_DC:
+        responses.add(
+            method=responses.GET, url=f"{DC_API_ENDPOINT}/workspaces/default/evaluation_sets", status=200, body=[]
+        )
+    else:
+        responses.add_passthru(DC_API_ENDPOINT)
+
+    with pytest.raises(DeepsetCloudError):
```

Comment:
Please add an expected error message pattern by specifying the `match` param of `pytest.raises`.